1.0.1/order-status.md

```diff
@@ -0,0 +1,2898 @@
+# Order Status Use Case
+
+## Context
+
+An _Order Issuer_ requests to a _Supplier_ the status of one or multiple of its _orders_. The _Order Issuer_ has earlier sent the orders to the Supplier. 
+This use case is designed for Pulp and Paper business and only for orders with direct shipments from the _Supplier_ site to the customer ship-to location(s).
```

Comment:
1/ I would again replace `orders` with `_orders_`.
2/ You mention the noun `customer` which I think we have never defined, could we replace it with `end user` that we are using the shipment status use case.
3/ Regarding `ship-to`, could we avoid that term since we are not (yet) defining it.